### PR TITLE
CHE-1095: add new UD/IDE navigation experience

### PR DESCRIPTION
**Pull request by akurinnoy**
Goals:
- revamp the left navbar
- remove the mobile responsiveness
- add ability to jump into recent workspace
- improve the branding/consistency between UD/IDE

Signed-off-by: Oleksii Kurinnyi <okurinnyi@codenvy.com>

### CODE REVIEW

**Comment by akurinnoy** [comment]
navbar-codenvy

**Comment by benoitf** [comment]
requiring https://github.com/eclipse/che/pull/1508

File: dashboard/src/app/navbar/navbar.controller.js [hunk 1]
```diff
@@ -68,14 +72,12 @@ export class CodenvyNavBarCtrl {
       projects: '#/projects',
       workspaces: '#/workspaces',
       factories: '#/factories',
+      team: '#/team',
```
Comment (olexii4) [inline dashboard/src/app/navbar/navbar.controller.js]: This is mistake after merge. Am I right?

Comment (akurinnoy) [inline dashboard/src/app/navbar/navbar.controller.js]: yes, you're right.
fixed

**Pull request merged**